pkg/instaclustr/client.go

```diff
@@ -630,6 +630,137 @@ func (c *Client) CreatePeering(url string, peeringSpec any) (*clusterresourcesv1
 	return creationResponse, nil
 }
 
+func (c *Client) CreateAzureVNetPeering(peeringSpec *clusterresourcesv1beta1.AzureVNetPeeringSpec, cdcId string) (*clusterresourcesv1beta1.PeeringStatus, error) {
+	payload := &struct {
+		PeerSubnets            []string `json:"peerSubnets"`
+		PeerResourceGroup      string   `json:"peerResourceGroup"`
+		PeerSubscriptionID     string   `json:"peerSubscriptionId"`
+		PeerADObjectID         string   `json:"peerAdObjectId,omitempty"`
+		PeerVirtualNetworkName string   `json:"peerVirtualNetworkName"`
+		CDCid                  string   `json:"cdcId"`
+	}{
+		PeerSubnets:            peeringSpec.PeerSubnets,
+		PeerResourceGroup:      peeringSpec.PeerResourceGroup,
+		PeerADObjectID:         peeringSpec.PeerADObjectID,
+		PeerSubscriptionID:     peeringSpec.PeerSubscriptionID,
+		PeerVirtualNetworkName: peeringSpec.PeerVirtualNetworkName,
+		CDCid:                  cdcId,
+	}
+
```

Comment:
peeringSpec alredy have cdcId as inline field. Why do we need to create new struct here?

Reply:
discussed